Description: CI - use correct mariadb packagenames
 Upstream uses the old names for libmariadb-dev-compat package.
Author: Craig Small <csmall@debian.org>
Origin: Debian
Bug-Debian: https://bugs.debian.org/975477
Reviewed-by: Craig Small <csmall@debian.org>
Last-Update: 2021-09-20
--- a/ci/before.sh
+++ b/ci/before.sh
@@ -7,7 +7,7 @@
 
 case "${TRAVIS_OS_NAME}" in
     linux)
-	sudo sh -c 'apt-get install -y libmariadbclient-dev || sudo apt-get install -y libmariadb-client-lgpl-dev'
+	sudo sh -c 'apt-get install -y libmariadb-dev-compat'
 
 	# Add an IPv6 config - see the corresponding Travis issue
 	# https://github.com/travis-ci/travis-ci/issues/8361
--- a/ci/net-snmp-configure
+++ b/ci/net-snmp-configure
@@ -52,7 +52,7 @@
                   grep -v 'not installed';
               done |
               head -n1;
-	      for p in libmariadbclient-dev libmariadb-client-lgpl-dev; do
+	      for p in libmariadb-dev-compat; do
 	        dpkg-query --show -f '${Version}\n' $p 2>/dev/null;
               done | grep -v '^$';
               pacman -Q mingw-w64-x86_64-libmariadbclient 2>/dev/null |
